Tests/Brokerages/Exante/ExanteBrokerageTests.cs

```diff
@@ -0,0 +1,75 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.Brokerages.Exante;
+using QuantConnect.Interfaces;
+using QuantConnect.Securities;
+using QuantConnect.Configuration;
+using QuantConnect.Lean.Engine.DataFeeds;
+using System.Linq;
+
+namespace QuantConnect.Tests.Brokerages.Exante
+{
+    [TestFixture]
+    public partial class ExanteBrokerageTests : BrokerageTests
```

Comment:
This is still to be implemented right?

Reply:
Some tests are derived from the base class. Some are added. All pass. I'll check from other classes what do you test. But if you comment on your requirements for tests, it would help.

Reply:
@Martin-Molinero :arrow_up: ?

Reply:
We've updated the https://github.com/QuantConnect/Lean.Brokerages.Template/tree/master/QuantConnect.TemplateBrokerage.Tests to include a nice set of expected tests (might depend on the brokerage implementation, supports IDQH etc)